README.md

```diff
@@ -7,6 +7,8 @@ This is part of [Integritee](https://integritee.network)
 ## Build and Run
 Please see our [Integritee Book](https://book.integritee.network/howto_worker.html) to learn how to build and run this.
 
+To start multiple worker and a node with one simple command: Check out [this README](https://github.com/integritee-network/worker/blob/master/local-setup/README.md).
+
```

Comment:
The link should be a relative path: `[this README](local-setup/README.md).`

Then you can also test it: :)

Reply:
Okay, I'll try that. Learned something new again. Thanks

Reply:
Crazy. It works!